src/SFA.DAS.EmployerFinance.sln

```diff
@@ -3,36 +3,32 @@ Microsoft Visual Studio Solution File, Format Version 12.00
 # Visual Studio 15
 VisualStudioVersion = 15.0.27703.1
 MinimumVisualStudioVersion = 10.0.40219.1
-Project("{2150E333-8FDC-42A3-9474-1A3956D46DE8}") = "Tests", "Tests", "{B9600BBE-2E67-4588-839E-C556C35CD682}"
```

Comment:
Can the `Tests` solution folder be added back in?

Reply:
sure

Reply:
done